public/cli/lib/commands/frames.js

```diff
@@ -168,10 +170,70 @@ function callRender(done) {
         .catch(done);
 }
 
+/**
+ * @function calculateRemainTime
+ * 
+ * @param {number} currFrame
+ * @param {number} totalFrame
+ * @return {string}
+ */
+function calculateRemainTime(currFrame, totalFrame) {
+
+    let result = "";
+    let currentDate = new Date();
+
+    // Skip calculating if it is the first run
+    if (lastUpdateFrameDate != null) {
+
+        // ((currentDate - lastUpdateFrameDate) / (currFrame - lastCurrentFrame)) * (totalFrame - currFrame)
+
+        let spendTime = currentDate - lastUpdateFrameDate; // milliseconds
+        let progressFrame = currFrame - lastCurrentFrame;
+        let spendTimePerFrame = spendTime / progressFrame;
+        let remainingFrames = totalFrame - currFrame;
+
+        let estimateTime = remainingFrames * spendTimePerFrame; // milliseconds
+
+        // Convert milliseconds to a readable string
+        let days = (estimateTime / 86400000).toFixed(0);
+        let hours = (estimateTime / 3600000).toFixed(0);
+        let minutes = (estimateTime / 60000).toFixed(0);
+        let seconds = (estimateTime / 1000).toFixed(0);
+
+        if (seconds < 1)
+            result = "";
+        else if (seconds < 60) 
+            result = `${seconds} second${seconds > 1 ? 's' : ''}`;
+        else if (minutes == 1)
+            result = `1 minute ${seconds - 60} seconds`;
+        else if (minutes < 60)
+            result = `${minutes} minutes`;
+        else if (hours < 24)
+            result = `${hours} hour${hours > 1 ? 's' : ''}`;
+        else
+            result = `${days} day${days > 1 ? 's' : ''}`;
+
+        if (result)
+            result = `(Remaining ${result})`;
```

Comment:
Lets change this to `(Approximately ${result} Remaining)`

Reply:
I updated it 
https://github.com/sillsdev/bible-karaoke/commit/625efd8a33b11f9a51e1bc01d360d7dff8e3d03f
https://github.com/sillsdev/bible-karaoke/commit/c640aee00ff0cdd5150c0200e6f18a56c9504305